kvmagent/kvmagent/test/localstorage_testsuite/test_data_volume_1.py

```
@@ -0,0 +1,116 @@
+            data_vol["installPath"] = vol_path
+            return data_vol
+
+        def build_virtio_scsi_vol(vol_uuid, vol_path):
+            data_vol = copy.deepcopy(start_vm_data_vol)
+            data_vol["volumeUuid"] = vol_uuid
+            data_vol["resourceUuid"] = vol_uuid
+            data_vol["installPath"] = vol_path
+            return data_vol
+
+        data_vol = copy.deepcopy(start_vm_data_vol)
+        data_vol["volumeUuid"] = vol_uuid
```

Ruff (0.12.2)

`vol_uuid` may be undefined, or defined from star imports

(F405)